tests/core/pyspec/eth2spec/test/deneb/block_processing/test_process_voluntary_exit.py

```diff
@@ -0,0 +1,45 @@
+from eth2spec.test.context import (
+    always_bls,
+    spec_state_test,
+    with_phases,
+    with_deneb_and_later,
+)
+from eth2spec.test.helpers.constants import (
+    DENEB,
+)
+from eth2spec.test.bellatrix.block_processing.test_process_voluntary_exit import (
+    run_voluntary_exit_processing_test,
+)
+
+
```

Comment:
We should add a test in which it is after the fork and `CAPELLA_FORK_VERSION` is used. This will just be a duplicate of `test_voluntary_exit_with_previous_fork_version_not_is_before_fork_epoch` for the Deneb for but it's important that it is tested and works for all subsequent forks

We also need a test for the fork *after* deneb to test that previous_fork_version doesn't work anymore, but we can't really do taht here. maybe we should make an issue?

Reply:
Ah, I see you did the latter with the 6110 tests which ... works but artificially makes 6110 a "fork" to capture this functionality. Probably fine

Still, I think we should write the explicit `CAPELLA_FORK_VERSION` test for deneb-and-after to catch it for all future forks

Reply:
Right, given that eip6110 is unstable, I should not have written it in eip6110. However, if I were to include it in a single test case, it would disrupt the test case naming convention: the prefix "invalid" is typically used to denote whether a state transition is valid or not. I have been back and forth on it... 😬